Gemfile

```diff
@@ -12,3 +12,7 @@ gem 'redcarpet'
 gem 'yard-activesupport-concern'
 
 gem 'solargraph'
+
+gem 'benchmark-ips'
+
+gem 'rspec-its'
```

Comment:
Since rspec-its is kinda dead, I'd maybe push to not use it if we don't have a really good reason to.